.github/workflows/ci-cd.yml

```
@@ -224,8 +244,9 @@ jobs:
               await github.rest.issues.createComment({ owner: ..., repo: ..., issue_number: pr.number, body: comment });
             }
             console.log(`Commented on PR #${pr.number}: ${pr.title}`);
+        env:
+          APP_VERSION: ${{ needs.deploy-development.outputs.APP_VERSION || needs.deploy-production.outputs.APP_VERSION }}
```

actionlint (1.7.12)

[error] property "app_version" is not defined in object type {}

(expression)